### PR TITLE
Feature/task3

### PR DESCRIPTION
**Pull request by katemalets**

### CODE REVIEW

**Review by nareykov — CHANGES_REQUESTED** [review]
Let's create a new branch for this task to keep the only task3-related class there

**Comment by katemalets** [comment]
deleted task 2 from this branch

File: ContactSearch.apxc [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+The Apex class must be called ContactSearch and be in the public scope
+The Apex class must have a public static method called searchForContacts
+The method must accept two incoming strings as parameters
+The method should then find any contact that has a last name matching the first string, and mailing postal code (API name: MailingPostalCode) matching the second string
+The method should finally return a list of Contact records of type List that includes the ID and Name fields
+*/
+
+public class ContactSearch {
+    public static List<Contact> searchForContacts(String lastName, String mailingPostalCode) {
+        List<Contact> contactList = [select ID, Name from Contact where (LastName = :lastName and MailingPostalCode = :mailingPostalCode)];
```
Comment (nareykov) [inline ContactSearch.apxc:13]: We prefer to write SOQL query using uppercase, like 'SELECT Id, Name FROM Contact WHERE...'

**Review by nareykov — APPROVED** [review]

File: ContactSearch.apxc (hunk 1 above)
Comment (katemalets) [reply ContactSearch.apxc:13]: Got it. Will do it in future.

**Pull request closed without merging**